packages/admin/src/utils/scrapperContext.ts

```diff
@@ -0,0 +1,23 @@
+import { createContext } from "react";
+
+export enum ScrapperConfig {
+  notice = "학과별 공지사항",
+  studentRestaurant = "힉생 식당",
+  domitoryRestaurant = "기숙사 식당",
+  collegeSchedule = "학사일정",
+}
```

Comment:
enum의 속성의 값은 `PascalCase`를 따르는것으로 합시다!
```suggestion
export enum ScrapperConfig {
  Notice = "학과별 공지사항",
  StudentRestaurant = "힉생 식당",
  DomitoryRestaurant = "기숙사 식당",
  CollegeSchedule = "학사일정",
}
```

Reply:
넵!